### PR TITLE
fixes #5934: changes needed to support creation of docker containers

### PR DESCRIPTION
**Pull request by dmitri-d**
- ssh provisioning step can be turned off
- os and architecture can be made optional

### CODE REVIEW

File: app/models/compute_resource.rb [hunk 1]
```diff
@@ -248,6 +248,18 @@ def user_data_supported?
     false
   end
 
+  def ssh_provisioning_supported?
+    true
```
Comment (ohadlevy) [inline app/models/compute_resource.rb:252]: this value depends on if image based provisioning is enabled and images has ssh flag on them.

**Comment by ohadlevy** [comment]
How would it look if we add another host type instead of changing the CR this way? I'm wondering if this is the right way forward long term?

**Comment by dLobatog** [comment]
This currently is outdated with the changes in the plugin that allow to create containers, mind if we close this?

**Comment by dmitri-d** [comment]
nah, not at all. closing.

**Pull request closed without merging**